api/src/test/java/org/pmiops/workbench/workspaceadmin/WorkspaceAdminControllerTest.java

```diff
@@ -192,126 +169,13 @@ public void setUp() {
-  }
-
-  @Test
-  public void getCloudStorageTraffic_sortsPointsByTimestamp() {
-    TimeSeries timeSeries =
-        TimeSeries.newBuilder()
-            .addPoints(
-                Point.newBuilder()
-                    .setInterval(TimeInterval.newBuilder().setEndTime(Timestamps.fromMillis(2000)))
-                    .setValue(TypedValue.newBuilder().setDoubleValue(1234)))
-            .addPoints(
-                Point.newBuilder()
-                    .setInterval(TimeInterval.newBuilder().setEndTime(Timestamps.fromMillis(1000)))
-                    .setValue(TypedValue.newBuilder().setDoubleValue(1234)))
-            .build();
-
-    when(mockCloudMonitoringService.getCloudStorageReceivedBytes(anyString(), any(Duration.class)))
-        .thenReturn(Arrays.asList(timeSeries));
-
-    CloudStorageTraffic cloudStorageTraffic =
-        workspaceAdminController.getCloudStorageTraffic(WORKSPACE_NAMESPACE).getBody();
-
-    assertThat(
-            cloudStorageTraffic.getReceivedBytes().stream()
-                .map(timeSeriesPoint -> timeSeriesPoint.getTimestamp())
-                .collect(Collectors.toList()))
-        .containsExactly(1000L, 2000L);
-  }
-
-  // TODO(jaycarlton) use  WorkspaceMapper.toDbWorkspace() once it's available RW 4803
-  private DbWorkspace createDbWorkspaceStub(Workspace workspace) {
-    DbWorkspace dbWorkspace = new DbWorkspace();
-    dbWorkspace.setWorkspaceId(DB_WORKSPACE_ID);
-    dbWorkspace.setName(workspace.getName());
-    dbWorkspace.setWorkspaceNamespace(workspace.getNamespace());
-    // a.k.a. FirecloudWorkspace.name
-    dbWorkspace.setFirecloudName(workspace.getId()); // DB_WORKSPACE_FIRECLOUD_NAME);
-    ResearchPurpose researchPurpose = workspace.getResearchPurpose();
-    dbWorkspace.setDiseaseFocusedResearch(researchPurpose.getDiseaseFocusedResearch());
-    dbWorkspace.setDiseaseOfFocus(researchPurpose.getDiseaseOfFocus());
-    dbWorkspace.setMethodsDevelopment(researchPurpose.getMethodsDevelopment());
-    dbWorkspace.setControlSet(researchPurpose.getControlSet());
-    dbWorkspace.setAncestry(researchPurpose.getAncestry());
-    dbWorkspace.setCommercialPurpose(researchPurpose.getCommercialPurpose());
-    dbWorkspace.setSocialBehavioral(researchPurpose.getSocialBehavioral());
-    dbWorkspace.setPopulationHealth(researchPurpose.getPopulationHealth());
-    dbWorkspace.setEducational(researchPurpose.getEducational());
-    dbWorkspace.setDrugDevelopment(researchPurpose.getDrugDevelopment());
-
-    dbWorkspace.setSpecificPopulationsEnum(new HashSet<>(researchPurpose.getPopulationDetails()));
-    dbWorkspace.setAdditionalNotes(researchPurpose.getAdditionalNotes());
-    dbWorkspace.setReasonForAllOfUs(researchPurpose.getReasonForAllOfUs());
-    dbWorkspace.setIntendedStudy(researchPurpose.getIntendedStudy());
-    dbWorkspace.setAnticipatedFindings(researchPurpose.getAnticipatedFindings());
-    return dbWorkspace;
-  }
-
-  @Test
-  public void testGetAuditLogEntries() {
```

Comment:
I killed these two, because it boiled down to `m = mock(); assert(m);` after the method moved.